client/checkout/woopay/express-button/woopay-express-checkout-payment-method.js

```diff
@@ -44,6 +44,9 @@ const WooPayExpressCheckoutButtonContainer = () => {
 
 const wooPayExpressCheckoutPaymentMethod = () => ( {
 	name: PAYMENT_METHOD_NAME_WOOPAY_EXPRESS_CHECKOUT,
+	title: 'WooPayments - WooPay',
+	description: 'Conditions for WooPay to be displayed',
```

Comment:
It would be good to use the description field to let merchants know what conditions need to be met in order for the buttons to be previewed. I don't know what these are for WooPay